src/main/thrift/parquet.thrift

```diff
@@ -174,7 +174,19 @@ enum ConvertedType {
    * particular timezone or date.
    */
   INTERVAL = 21;
-  
+
+  /**
+   * Annotates a column that is always null
+   * Sometimes when discovering the schema of existing data
+   * values are always null
+   */
+  NULL = 25;
+
+  /**
+   * Annotates a FLOAT type when the original type was half precision on 16 bits
+   * IEEE 754-2008 the 16-bit base 2 format
+   */
+  FLOAT_HALF_PRECISION = 26;
```

Comment:
As mentioned yesterday in the sync: This seems to be neither implemented in Arrow.